docassemble/HousingCodeChecklist/data/questions/housing_code_interview.yml

```
@@ -865,38 +866,42 @@ template: how_many_problems_are_there_template
 subject: |
   How many problems can you help me with?
 content: |
-  If you explore room by room, you can pick from a list of about
+  If you explore by room and category, you can pick from a list of about
   150 problems. Some may be problems you did not know your
   landlord needs to fix.
 
-  The list comes from the Massachusetts Sanitary Code. We have
+  The list of problems comes from the [State Sanitary Code](https://www.mass.gov/doc/105-cmr-410-state-sanitary-code-chapter-ii-minimum-standards-of-fitness-for-human-habitation/download). We have
```

```suggestion
  The list of problems comes from the State Sanitary Code. We have
```

We don't want someone to open a link to an external website at this point. We could add a glossary `{term}` if we want to provide more context here.